news/_posts/2017-07-20-slick-3.2.1-released.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: news
+title: Slick 3.2.1
+author: Trevor Sibanda
+---
+We have just released Slick 3.2.1.
+You can find the source code here: <https://github.com/slick/slick/tree/3.2.1>.
+Builds for Scala 2.11 and 2.12 are available from Maven Central, as usual.
+From 3.2.0 on, Slick requires Java 8. Older JVMs are no longer supported.
+
+This is the first version of Slick released from automated releases.
+
+##### These are the major changes in Slick 3.2.1:
+
+* [``fb9258a1c``](https://github.com/slick/slick/commit/fb9258a1c78255726a78f5cd8fd5320df05c6a77) fixes race-condition using unpinned/flatMapped DBIOActions, leading to missing decreaseInUseCount calls and possible stall of ManagedBlockingArrayQueue* Important bug fixes for asynchronous execution of database actions and other parts of
+  the codebase.
+* [``3728a2092``](https://github.com/slick/slick/commit/3728a2092d78afd53b54d0b9f6cfdf29af1fb3c5) regression: final actually breaks compilation
+* [``3f5c6753b``](https://github.com/slick/slick/commit/3f5c6753b4108564a3583c8b49f0a89374a6dd4c) fixing part 1 of #1408, flatmaps in the tail position of a query are unrolled into nested bind clauses* Strip margins of multi-line SQL interpolator strings.
+* [``8b342b4a1``](https://github.com/slick/slick/commit/8b342b4a177770431223c2a2559041a009eda9f3) Integrate the four official Activator templates into the manual.
+* [``ddb75a0f9``](https://github.com/slick/slick/commit/ddb75a0f96fdde48bbb6885c4636bae204515a69) Fix ambiguity for MappedProjection Shape
+* [``21045bbf2``](https://github.com/slick/slick/commit/21045bbf2df79f18feabe0c147dfb0dc202f86a9) use mix of Java/scala reflection to retrieve inUseCount on 2.12.x
+
+
+You can find details about upgrading from Slick 3.2 in the *Migration Guide*. 
```

Comment:
...oh, and it's called *Upgrad Guide* in the actual manual. If you search for "migration" you get something about database migrations.